src/tests/Misc/Drawing.cs

```diff
@@ -156,6 +157,12 @@ public void Test_LettersDontRenderAsRectangles_SerifFont()
             // this test ensures letters don't render as rectangles
             // https://github.com/ScottPlot/ScottPlot/issues/1079
 
+            // https://github.com/ScottPlot/ScottPlot/pull/1310#issuecomment-932638916
+            // Some potential change to the default fonts provided for the linux image,
+            // will still run through the test on the windows portion of CI
+            if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
```

Comment:
This is dicey, but I think if we have to not run it on Linux we should probably do this instead:

```cs
if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
```

Reply:
Good catch! However I'll revert this test to its original form and, now that the correct version of System.Drawing.Common is loaded, the test passes again on all operating systems.

Reply:
c34cd0c